tests/functional/columns_in_relation_tests/test_columns_in_relation.py

```diff
@@ -28,6 +24,10 @@ def test_columns_in_relation(self, project, expected_columns):
         )
         with project.adapter.connection_named("_test"):
             actual_columns = project.adapter.get_columns_in_relation(my_relation)
+        expected_columns = [
```

Comment:
Since we are aliasing dtypes back to the original names, we don't need to specify the expected column dtypes based on whether the flag is enabled.